recipes/billingegroup-meta/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "billingegroup-meta" %}
+{% set version = "0.1.0" %}
```

Comment:
We make heavy use of metapackages at my current employer and have found that three version numbers is overkill. Honestly two is probably more than is needed. Unless you really need two or three version numbers, I would recommend sticking to just a single number. i.e., `{% set version = 1 %}`